crates/swc_ecma_transforms_proposal/src/decorators/legacy/mod.rs

```diff
@@ -36,6 +36,8 @@ pub(super) fn new(metadata: bool, use_define_for_class_fields: bool) -> TscDecor
 
 pub(super) struct TscDecorator {
     metadata: bool,
+
+    #[allow(dead_code)]
     use_define_for_class_fields: bool,
```

Comment:
We should leave `class fields` to be handled in `swc_ecma_transforms_compat::es2022::class_properties` stage, not `decorators`.

This `use_define_for_class_fields` option is not used in this transformation.
Is it safe to remove it wihout breaking changes?

Reply:
I think so

Reply:
It's passed, not deserialized from json